test/component/hooks/sn.comp.js

```diff
@@ -71,7 +73,7 @@ describe('hooks', () => {
 
   it('useConstraints', async () => {
     const text = await page.$eval(`${snSelector} .constraints`, (el) => el.textContent);
-    expect(text).to.equal('false:false:true');
+    expect(text).to.equal('false:false:false');
```

Comment:
I'm just curious to why this has changed from `false:false:true` into `false:false:false`... Can't find the reason for it..

Reply:
With the the previous mocking solution the `select` constraint was by default set to `true`. The difference between the previous and `EnigmaMocker` is that a generic object is supplied, rather than using `createSessionObject()` to instantiate objects. I suspect this is the reason why it differs.

I have verified that it's possible to set constraints in the fixtures.